code/modules/antagonists/nukeop/equipment/nuclearbomb.dm

```diff
@@ -511,9 +513,25 @@ This is here to make the tiles around the station mininuke change when it's arme
 	else
 		GLOB.poi_list |= src
 		tell_the_admins = TRUE
+		START_PROCESSING(SSobj, src)
 
 	set_stationloving(TRUE, inform_admins=tell_the_admins)
 
+/obj/item/disk/nuclear/process()
+	if(fake)
+		STOP_PROCESSING(SSobj, src)
+		return
```

Comment:
521-23 are unneeded (see 513)

Reply:
It's mainly a failsafe just in case a fake nuke disk *somehow* starts processing. Making code that's paranoid about edge cases is always a good idea

Reply:
you're hiding the bug then.